packages/date-picker/src/vaadin-date-picker-light.js

```diff
@@ -74,7 +74,7 @@ class DatePickerLight extends ThemableMixin(DatePickerMixin(ValidateMixin(Polyme
         fullscreen$="[[_fullscreen]]"
         opened="{{opened}}"
         on-vaadin-overlay-open="_onOverlayOpened"
-        on-vaadin-overlay-close="_onOverlayClosed"
+        on-vaadin-overlay-closing="_onOverlayClosed"
```

Comment:
```suggestion
        on-vaadin-overlay-closing="_onOverlayClosing"
```

Reply:
This method is marked as protected in JSDoc, that's why I didn't want to rename it.
In theory, it might be used by classes extending `vaadin-date-picker` (or `-light`).

Reply:
Ah, right, let's not touch it then.